### PR TITLE
ensure inserting to correct container

### PR DESCRIPTION
**Pull request by mmacy**
Ensure that we're inserting into the 'products' container, with a couple doc updates tacked on.

### CODE REVIEW

**Comment by mmacy** [comment]
ping @johanste

File: README.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Once you've initialized a [CosmosClient][ref_cosmosclient], you can interact wit
 
 * [Container][ref_container]: A container is a collection of JSON documents. You create (insert), read, update, and delete items in a container by using methods on the [Container][ref_container] object.
 
-* [Item][ref_item]: An Item is the dictionary-like representation of a JSON document stored in a container. Each Item you add to a container must have an `id` field whose value is unique within the container. If you add an item to a container without specifying a unique ID, Cosmos DB creates one for you in the form of a GUID.
+* [Item][ref_item]: An Item is the dictionary-like representation of a JSON document stored in a container. Each Item you add to a container must have an `id` field whose value is unique within the container. If you add an item to a container without including an `id` field, Cosmos DB creates one for you in the form of a GUID.
```
Comment (johanste) [inline README.md]: Automatic ID creation has been turned off.

Comment (mmacy) [reply README.md]: D'oh! OK, there are several edits that are needed, in that case.

Comment (mmacy) [reply README.md]: How's that look, @johanste?

File: README.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ Once you've initialized a [CosmosClient][ref_cosmosclient], you can interact wit
 
 * [Container][ref_container]: A container is a collection of JSON documents. You create (insert), read, update, and delete items in a container by using methods on the [Container][ref_container] object.
 
-* [Item][ref_item]: An Item is the dictionary-like representation of a JSON document stored in a container. Each Item you add to a container must have an `id` field whose value is unique within the container. If you add an item to a container without specifying a unique ID, Cosmos DB creates one for you in the form of a GUID.
+* [Item][ref_item]: An Item is the dictionary-like representation of a JSON document stored in a container. Each Item you add to a container must include an `id` field whose value is unique within the container.
```
Comment (johanste) [inline README.md]: Maybe: 

an `id` field that uniquely identifies the item within the container?

Also, technically, it is not a field, but a key-value pair (which is sometimes referred to as an item in python's dict documentation). So maybe:

an item with the key `id` that uniquely identifies the item within the collection. 

?

Comment (mmacy) [reply README.md]: How about:

Each Item you add to a container must include an `id` key with a value that's unique within the container.

...or a little more explicit on the "dictness" of the Item so people more consciously note the *key*  and *value* keywords:

Each Item dict you add to a container must include an `id` key with a value that's unique within the container.

Comment (mmacy) [reply README.md]: Or this, which uses your "uniquely identifies" term:

Each Item you add to a container must include an `id` key with a value that uniquely identifies it within the container.

Comment (mmacy) [reply README.md]: @johanste Updated! Lemme know what you think.

File: README.md (hunk 1 above)
Comment (johanste) [reply README.md]: The same comment about `id` field phrasing as below.

**Review by johanste — APPROVED** [review]

**Pull request merged**